### PR TITLE
Feature 

### PR DESCRIPTION
**Pull request by mutinhiri**
in this PR i set up the web using webpack and added setup for ongoing leaderboard project

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        </div>
+        <div class="add">
+            <p>add your score</p>
+            <div class="input-items">
+                <input type="text" placeholder="Your name">
+                <input type="text" placeholder="Your score">
+                <button id="submit">Submit</button>
+            </div>
+           
+        </div>
```
Comment (henatan99) [inline src/index.html:31]: - [x] I think this project is about applying JS DOM with ES6 syntaxes to dynamically create elements. I recommend you to generate these elements dynamically in your js file/s, with proper ES6 syntax. :pray:

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import './style.css';
+import './dom.js';
+
+const dom = document.getElementById('dom');
+
+dom.innerHTML = '<li>webpack setup</li>';
```
Comment (henatan99) [inline src/index.js]: - [x] There is no HTML element with id 'dom'. As a result, the app throws a browser console error. Kindly fix this. You can use this file to create elements dynamically :pray: 

Screenshot from 2021-11-17 00-37-31

File: src/dom.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const submit = document.getElementById('submit');
+
+function runEvent() {
+  return true;
+}
+
+submit.addEventListener('click', runEvent);
```
Comment (henatan99) [inline src/dom.js]: - [x] This element is not exported, so it is an unused module.  Kindly delete it or use it by exporting it from here and importing it from the parent element module file. :pray: 
- [x] Also, kindly use ES6 syntax for function definition (arrow function) as it is the project's objective to practice ES6 syntax. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @mutinhiri , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @mutinhiri,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**